python/pyiceberg/catalog/__init__.py

```diff
@@ -238,8 +238,8 @@ class Catalog(ABC):
     or tuple of strings.
 
     Attributes:
-        name (str | None): Name of the catalog
-        properties (Properties): Catalog properties
+        name (str | None): Name of the catalog.
```

Comment:
Just noticed this now:
```suggestion
        name (str): Name of the catalog.
```